metricbeat/module/graphite/_meta/config.reference.yml

```diff
@@ -0,0 +1,10 @@
+- module: graphite
+  metricsets: ["server"]
+  enabled: true
+  #protocol: "udp"
```

Comment:
Should we add an example of how to configure listening address and port?

Reply:
Good call. I added quite a bit more details and config parameters. Graphite will need more docs later.